src/utils/question.rs

```diff
@@ -48,49 +48,71 @@ pub enum FileConflitOperation {
     /// Rename the file
     /// It'll be put "_1" at the end of the filename or "_2","_3","_4".. if already exists
     Rename,
+    /// Merge duplicated files
+    Merge,
```

Comment:
We have a flag for always overwriting and a flag for always renaming, can you add a flag `--merge` to always merge? (without asking the prompt).

Reply:
Could you guide me on which flag represents the always renaming?
because, from my understanding
```bash
flag --yes = always override
```
And yes, I can add a new flag in the decompress command to always merge.

Reply:
Huh I thought the rename PR added a flag for that...

Reply:
hmmm, sorry, I read the question.rs deeply found out to add a new flag --merge we need to refactor the function `pub fn user_wants_to_overwrite` which might be beyond the PR's scope.

Reply:
I think these would be a good addition, these would be welcome for another PR :) primarily the `--merge`, right now we have `--overwrite` in the form of `--yes`, which I think should be adjusted now that we have more than `yes` and `no` questions.

I think `--yes` shouldn't overwrite, so for folks to use this in scripts, they can provide `--yes` for the `y/n` questions and `--overwrite` for file conflicts.